### ISSUE TITLE
Trying to read parquet file from python (pandas, pyarrow) result in a `stream-rows` length

### ISSUE THREAD

**Issue by ant1j** [curtisalexander/readstat-rs#63]
I tried your excellent & fast tool to convert sas7bdat to parquet file.

Once converted, and trying to use the parquet file in python pandas, the length of the resulted dataframe is only `stream-rows` (I tryed 10k and 100k).

Using `pyarrow` as well with `pyarrow.RecordBatch`, I can only "see" the same length of records.

=> Am I missing something here?

```python
from pyarrow.parquet import ParquetFile

# original sas7bdat file is 3.7m+ rows
# Parquet file is 387MB -- so holding more than 100k rows potentially
pf = ParquetFile("./rust_sales_b1.parquet")  

In: pf.scan_contents()
Out: 100000

In [8]: for b in pf.iter_batches():
   ...:     print(len(b))
65536
34464
```

Trying to consider the file as a pyarrow.dataset (partitioned parquet) result in same output.

**Comment by curtisalexander** [comment]
@ant1j I believe I had a bug in my code where I was re-opening a new parquet writer for every batch of rows.  Thus the size of the parquet file would only ever be the size of `--stream-rows`.

In fact, I had originally protected against this but commented out the code &mdash; most likely during some testing I was doing and never uncommented.

I made the correction and binaries are being built now in the `parquet-writer` branch.  I'll merge into `main` once they are fully built.

**Comment by curtisalexander** [comment]
I have no clue if the musl build will finish...feel free to try out one of the binaries under 0.9.3.

https://github.com/curtisalexander/readstat-rs/releases/tag/0.9.3

**Comment by ant1j** [comment]
Seems to be all good now. 
Thanks

```python 
In [3]: df = pd.read_parquet("./rust_sales_b0_2.parquet")

In [4]: df.shape
Out[4]: (3806096, 31)
```

**Comment by curtisalexander** [comment]
Great news!  Closed with 378accd.
